0001-encoder_h264-add-parameter-Cabac-Dct8x8-and-DeblockF.patch

```diff
@@ -0,0 +1,254 @@
+From 905a0d05a83dfd2f4f0e994ff4cc628dd1d708e1 Mon Sep 17 00:00:00 2001
```

Comment:
;), maybe you need 
`git commit -a -m "comments"`
to submit your changes.
Hope you can use "git show" before you do any git push.